contracts/BridgeLimiter.sol

```diff
@@ -75,24 +79,30 @@ contract BridgeLimiter is
 
     /* ========== EXTERNAL FUNCTIONS ========== */
 
+    // there are duplicated calls to `calculateAmountInUSD`, one in `willAmountExceedLimit`
+    // and one in in line 100. Can we modify `willAmountExceedLimit` to let it take the USD amount instead?
     function updateBridgeTransfers(uint8 tokenId, uint256 amount) external override onlyOwner {
         require(amount > 0, "BridgeLimiter: amount must be greater than 0");
         require(
             !willAmountExceedLimit(tokenId, amount),
             "BridgeLimiter: amount exceeds rolling window limit"
         );
 
+        // We have this in a few places as well
         uint32 currentHour = uint32(block.timestamp / 1 hours);
 
         // garbage collect most recently expired hour if window is moving
+        // why do we need `hourlyTransferAmount[currentHour] == 0`?
         if (hourlyTransferAmount[currentHour] == 0 && oldestHourTimestamp < currentHour - 24) {
+            // are the parameters correct?
             garbageCollectHourlyTransferAmount(currentHour - 25, currentHour - 25);
         }
 
         // update hourly transfers
         hourlyTransferAmount[currentHour] += calculateAmountInUSD(tokenId, amount);
     }
 
+    // I don't understand this function, primarily because I don't understand the args passed in a few lines above
```

Comment:
name needs to be more explicit (and some documentation wouldn't hurt lol). this function exists to enable the removal of expired hourly transfers (transfers older than 24 hours). the parameters resemble a range of hours to iterate through to try and remove from storage.

Reply:
extract 
```
        for (uint32 i = startHour; i <= endHour; i++) {
            if (hourlyTransferAmount[i] > 0) delete hourlyTransferAmount[i];
        }
        // update oldest hour if current oldest hour was garbage collected
        if (startHour == oldestHourTimestamp) {
            oldestHourTimestamp = endHour + 1;
        }
```
to its own function